source/scale/high-availability-cluster-based-deployment.rst

```diff
@@ -282,7 +282,7 @@ On large deployments, also consider using the :ref:`search replicas <configure/e
 
 .. note::
 
-  For an AWS High Availability RDS cluster deployment, don't hard-code the IP addresses. Point this configuration setting directly to the underlying read-only node endpoints within the RDS cluster. We recommend circumventing the failover/load balancing that AWS/RDS takes care of (except for the write traffic). Mattermost has its own method of balancing the read-only connections, and can also balance those queries to the DataSource/write+read connection should those nodes fail. 
+  For an AWS High Availability RDS cluster deployment, don't hard-code the IP addresses. Point this configuration setting directly to the underlying read-only node endpoints within the RDS cluster. We recommend circumventing the failover/load balancing that AWS/RDS takes care of (except for the write traffic), and populate the `DataSourceReplicas` array with the RDS reader node endpoints. Mattermost has its own method of balancing the read-only connections, and can also balance those queries to the DataSource/write+read connection should those nodes fail.
```

Comment:
Should we address the other issue with `DataSourceReplicas` env override? I recall it wasn't clear it needed to be a space-delimited list, so the customer was left guessing. In https://docs.mattermost.com/configure/environment-configuration-settings.html#read-replicas I mean.

Reply:
I think we should treat that separately. It is applicable to any array. Maybe we can mention that in the config section somewhere above.

Reply:
Up to you. It just seems like more overhead to open a new PR vs editing a line in a file.

Reply:
The env var ambiguity re: ``DataSourceReplicas`` is addressed via https://github.com/mattermost/docs/pull/7873 scheduled for release next week with v10.7